docs/gene_sets.md

```diff
@@ -0,0 +1,103 @@
+## Overview
+
+Users may view gene sets along with their dataset and access gene sets created in pip installed cellxgene by following the instructions below.
+
+The order of the gene symbols in each gene set is maintained in the database. 
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED" "MAY", and "OPTIONAL" in this document are to be interpreted as described in BCP 14, [RFC2119](https://www.rfc-editor.org/rfc/rfc2119.txt), and [RFC8174](https://www.rfc-editor.org/rfc/rfc8174.txt) when, and only when, they appear in all capitals, as shown here.
+
+## cellxgene CLI commands
+
+Users MAY use --user-generated-data-dir `file_name` to designate a file where both gene sets and annotations CSVs will be saved.
+
+Users MAY use --gene-sets-file `name_of_file.csv` to designate a CSV with preexisting gene sets you would like to view in cellxgene. The CSV MUST follow the format below.
+
+## cellxgene gene set CSV data format
+  
+The cellxgene gene set data format is a [*Tidy* CSV](./gene_sets_example.csv) (comma-separated values) file using ASCII encoding. Multiple gene sets MAY be included in the file similar to the [Gene Matrix Transposed](https://software.broadinstitute.org/cancer/software/gsea/wiki/index.php/Data_formats#GMT:_Gene_Matrix_Transposed_file_format_.28.2A.gmt.29) or [Gene Matrix](https://software.broadinstitute.org/cancer/software/gsea/wiki/index.php/Data_formats#GMX:_Gene_MatriX_file_format_.28.2A.gmx.29) formats.
+
+Example:
```

Comment:
I find the extraneous (non-normative) columns confusing.  I would keep your initial example minimal (eg, delete `provenance1...`).  If you feel that the prose in the Mandatory Header section needs further examples to highlight the allowance for "additional columns", add it there.